learn/user-guide/why-ballerina/network-aware-type-system.md

```diff
@@ -63,7 +63,7 @@ A type in Ballerina represents a set of the possible shapes it can have. So any
 
 ### Subtypes in Ballerina
 
-Subtyping in Ballerina is semantic. It is defined by means of shapes, where S is a subtype of T, if the shapes representing S are a subset of the shapes representing T. Let’s demonstrate this behavior with a few examples.
+Subtyping in Ballerina is semantic. It is defined by means of shapes, where S is a subtype of T, if the set of shapes denoted by S are a subset of the shapes denoted by T. Let’s demonstrate this behavior with a few examples.
 
 The type `boolean` is a simple basic type in Ballerina without a storage identity, so its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes true and false.
```

Comment:
```suggestion
The type `boolean` is a simple basic type in Ballerina without a storage identity so that its values become equivalent to its shapes. Therefore, the `boolean` type is defined as having two shapes as `true` and `false`.
```

Reply:
Would `defined as having two shapes, true and false` work here?

Reply:
+1 Please add as `true and `false` within backticks.